content/en/docs/tasks/traffic-management/egress/egress-control/index.md

```diff
@@ -494,3 +494,40 @@ Shutdown the [sleep]({{< github_tree >}}/samples/sleep) service:
 {{< text bash >}}
 $ kubectl delete -f @samples/sleep/sleep.yaml@
 {{< /text >}}
+
+### Set the outbound traffic policy mode to your desired value
+
+1.  Check the current value:
+
+    {{< text bash >}}
+    $ kubectl get configmap istio -n istio-system -o yaml | grep -o "mode: ALLOW_ANY"
+    $ kubectl get configmap istio -n istio-system -o yaml | grep -o "mode: REGISTRY_ONLY"
+    mode: ALLOW_ANY
+    mode: ALLOW_ANY
```

Comment:
This is confusing. Is this grep command always going to show mode: xxx twice? I think one is the previous value in the metadata, right?

Reply:
The grep command shows mode: xxx twice. One of the values is `kubectl.kubernetes.io/last-applied-configuration`. I will just add `| uniq` so the value will be printed once.

Reply:
@frankbu Addressed in https://github.com/istio/istio.io/pull/5264/commits/80cf9503f50dbdc77c7f9d1d092dd807ad5ca04a.